WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingViewModel.swift

```diff
@@ -68,7 +68,7 @@ final class BlazeAdDestinationSettingViewModel: ObservableObject {
                     // Once a parameter is updated, clear the selected parameter to prepare for the next add/update action.
                     clearSelectedParameter()
                 } else {
-                    self.parameters.append(BlazeAdURLParameter(key: key, value: value))
+                    self.addNewParameter(item: BlazeAdURLParameter(key: key, value: value))
```

Comment:
Super nit: `self` can be removed.

Reply:
Done in 80b49327907ce25b5c44609254f9cdb6e7f61be6.